lib/active_attr/typecasting.rb

```diff
@@ -51,16 +51,16 @@ def typecast_attribute(typecaster, value)
       typecaster.call(value)
     end
 
-    # Resolve a Class to a typecaster
+    # Resolve an Instance to a typecaster
     #
     # @param [Class] type The type to cast to
     #
-    # @return [#call, nil] The typecaster to use
+    # @return [#call, nil] The typecaster instance to use
     #
     # @since 0.6.0
     def typecaster_for(type)
       typecaster = TYPECASTER_MAP[type]
-      typecaster.new if typecaster
+      typecaster if typecaster
```

Comment:
This final line is no longer necessary since you're not calling anything. The method simply becomes:

```
def typecaster_for(type)
    TYPECASTER_MAP[type]
end
```